backend/common/src/main/java/ai/verta/modeldb/common/futures/FutureExecutor.java

```diff
@@ -1,23 +1,21 @@
 package ai.verta.modeldb.common.futures;
 
-import io.grpc.Context;
-import io.opentracing.Scope;
-import io.opentracing.util.GlobalTracer;
+import io.opentelemetry.context.Context;
 import java.util.concurrent.Executor;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ForkJoinPool;
 import org.springframework.lang.NonNull;
 
 public class FutureExecutor implements Executor {
-  final Executor other;
+  private final Executor delegate;
 
-  FutureExecutor(Executor other) {
-    this.other = io.opentelemetry.context.Context.taskWrapping(other);
+  private FutureExecutor(Executor delegate) {
+    this.delegate = Context.taskWrapping(delegate);
```

Comment:
nit: I personally find very confusing to just have `Context` for otel given that +80% of the times we're using it for the grpc context. So my brain just assumes this is talking about the grpc context

Reply:
heh. Of course, I have the opposite confusion, having lived OTel for so long. I actually liked your solution I saw in another PR to just fully qualify both of them. I'll update!

Reply:
or...I'll wait for your changes to this to land, and just use them as-is.

Reply:
done!